CleanWeather/Scenes/CityDetails/CityDetailsDisplayable/CityDetailsDisplayable.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CityDetailsDisplayable.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 20/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct CityDetailsDisplayable {
+    
+    let id: String
+    let name: String
+    let temp: String
+    let symbol: WeatherSymbol
+    
+    init(object: CityWeather) {
+        id = object.id
+        name = object.city
+        temp = "\(Int(object.temperature.rounded()))°"
```

Comment:
you're using this code for the 2nd time, worth thinking about a helper that is converting Double to TemperatureString